### ISSUE TITLE
Question: Using find to making heatmaps of inside location

### ISSUE THREAD

**Issue by DomKarb** [schollz/find#187]
Hi,

I'm looking for solution to making a heatmap inside the location with data from find.
Do you know any solution to that?
I have idea to doing it myself, but don't want doing something if maybe someone did it already ;)
Looking forward.

Best regards
Dominik Karbowski

**Comment by schollz** [comment]
@DomKarb As far as I know this hasn't been done yet!

**Comment by DomKarb** [comment]
@schollz is it something your started end didn't finish yet?
https://github.com/schollz/findmap

**Comment by schollz** [comment]
Yeah I'm not working on that right now. Feel free to use it if youd like
